Command/CleanMediaCommand.php

```diff
@@ -88,26 +83,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
-     * @return string[]
-     */
-    private function getProviders()
-    {
-        if (!$this->providers) {
-            $this->providers = array();
-
-            $pool = $this->getContainer()->get('sonata.media.pool');
-
-            foreach ($pool->getProviders() as $provider) {
-                if ($provider instanceof FileProvider) {
```

Comment:
I guess this restriction is filtering to only keep FileProviders. This could be useful in case:

Have a file on the filesystem and when you findBy you get a Youtube (does not extends from FileProvider) with the same providerReference.

Maybe it needs to be kept?